tests/v2/validation/v2prov/encryption_key_rotation_test.go

```diff
@@ -169,17 +201,49 @@ func (r *V2ProvEncryptionKeyRotationTestSuite) TestEncryptionKeyRotation() {
 
 	setEncryptSecret(r.T(), r.client, id)
 
-	prefix := "encryption-key-rotation-"
-	r.Run(prefix+"new-cluster", func() {
-		rotateEncryptionKeys(r.T(), r.client, id, 1, 10*time.Minute)
+	clusterType, err := getClusterType(r.client, id)
+	require.NoError(r.T(), err)
+	prefix := clusterType + "-encryption-key-rotation"
+
+	r.Run(prefix, func() {
+		rotateEncryptionKeys(r.T(), r.client, id, 10*time.Minute)
+	})
+
+	if r.client.Flags.GetValue(environmentflag.Long) {
+		// create 10k secrets for stress test, takes ~30 minutes
+		createSecretsForCluster(r.T(), r.client, id, totalSecrets)
+
+		r.Run(prefix+"stress-test", func() {
+			rotateEncryptionKeys(r.T(), r.client, id, 1*time.Hour) // takes ~45 minutes for HA
+		})
+	}
+}
+
+func (r *V2ProvEncryptionKeyRotationTestSuite) TestEncryptionKeyRotationWithVaiEnabled() {
+	subSession := r.session.NewSession()
+	defer subSession.Cleanup()
+
+	id, err := clusters.GetV1ProvisioningClusterByName(r.client, r.clusterName)
+	require.NoError(r.T(), err)
+
+	setEncryptSecret(r.T(), r.client, id)
+	err = vai.DisableVaiCaching(r.client)
+	require.NoError(r.T(), err)
+
+	clusterType, err := getClusterType(r.client, id)
+	require.NoError(r.T(), err)
+	prefix := clusterType + "encryption-key-rotation-vai-enabled"
```

Comment:
add a `-` to `encryption...` for name formatting in qase

Reply:
done.